app/views/recipes/_form.html.haml

```diff
@@ -8,6 +8,9 @@
   - Recipe::TIME_ATTRIBUTES.each do |time_attribute|
     = f.input time_attribute, label: "#{Recipe.human_attribute_name time_attribute} (in Minutes)"
   
+  %h2 Notes
+  = f.input :notes, label: false
+  
   %h2 Ingredients
   #ingredients
```

Comment:
I was going to remove those divs but then changed my mind again.  They are grouping the normal form entries and the add button into one element.  That's also why we only have them for ingredients, directions and references.  The other form elements do not have an add button.